docs/eos-learn/ricardian-contracts.md

```diff
@@ -4,20 +4,59 @@ title: Ricardian contracts
 sidebar_label: Ricardian contracts
 ---
 
-## **What are Ricardian contracts?**
+## What are Ricardian Contracts?
 They are digital documents that define the terms and conditions among those involved in the contract, these are signed and verified cryptographically and are readable by both humans and computer systems.
 
-### **Characteristics of these contracts**
+### Characteristics of these contracts
 
 - Human and software readable.
 - It can be printed.
 - All forms of documents (physical, virtual, encrypted) are equivalent.
 - They are signed by all parties.
 - They can be identified safely thanks to the has of each contract.
 
-### **Ricardian contracts at EOSIO**
+### Ricardian contracts at EOSIO
 
 Blockchains operate under smart contracts, however, work is underway to include Ricardian contracts in the transactions of these networks. One of the main objectives is to convert the complexity of the code observed within Blockchain technology into a simpler language so that anyone could understand it.
```

Comment:
lets include a link to view an example Ricardian contract on our components storybook:

https://components.eosio.cr/?path=/story/ricardiancontract--ricardian-contract